tools/test/test.sh

```diff
@@ -34,6 +34,13 @@ elif [ "$SUITE" == "docs" ]; then
     tools/generate_doc.py
     test -s $RUCIO_CLIENT_API_OUTPUT
     test -s $RUCIO_REST_API_OUTPUT
+    tools/generate_rest_api_doc.py > api_doc.yaml
+
+    echo "Node version:"
+    node -v
+    npx @redocly/openapi-cli lint api_doc.yaml || true
```

Comment:
I think it would be nice if we can also run these checks locally ( like the add-header). Can we refactor this PR so it works with both the CI and for local runs? 

I propose that in the python script, we check 
1. if `node` is installed on the machine
2. if the `@redocly/openapi-cli` package is available
3. invoke the `@redocly/openapi-cli` as a python subprocess
4. check and print the output and do a sys.exit(0 or 1) accordingly.